pkg/ebpftracer/signature/git_clone_detected.go

```diff
@@ -0,0 +1,140 @@
+package signature
+
+import (
+	"strings"
+
+	v1 "github.com/castai/kvisor/api/v1/runtime"
+	"github.com/castai/kvisor/pkg/ebpftracer/events"
+	"github.com/castai/kvisor/pkg/ebpftracer/types"
+	"github.com/castai/kvisor/pkg/logging"
+)
+
+var _ Signature = (*GitCloneDetected)(nil)
+
+type GitCloneDetected struct {
+	log *logging.Logger
+}
+
+func NewGitCloneDetectedSignature(log *logging.Logger) Signature {
```

Comment:
nit: there's not much value in returning an interface from a function that returns a concrete type (and you already have type-assertion at the beginning of the file). AFAIK the recommendation in this case is to return the specific type.